plugins/output-kafka/src/main/scala/com/stratio/sparkta/plugin/output/kafka/KafkaOutput.scala

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (C) 2015 Stratio (http://stratio.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.stratio.sparkta.plugin.output.kafka
+
+import java.io.{Serializable => JSerializable}
+
+import com.stratio.sparkta.plugin.output.kafka.producer.KafkaProducer
+import com.stratio.sparkta.sdk.TypeOp.TypeOp
+import com.stratio.sparkta.sdk.WriteOp.WriteOp
+import com.stratio.sparkta.sdk._
+import org.apache.spark.sql._
+
+
+class KafkaOutput(keyName: String,
+                  version: Option[Int],
+                  properties: Map[String, JSerializable],
+                  operationTypes: Option[Map[String, (WriteOp, TypeOp)]],
+                  bcSchema: Option[Seq[TableSchema]])
+  extends Output(keyName, version, properties, operationTypes, bcSchema) with KafkaProducer {
+
+  override def upsert(dataFrame: DataFrame, tableName: String, timeDimension: String): Unit = {
+    dataFrame.toJSON.foreachPartition { messages => {
+      send(properties, tableName, messages.toList.mkString)
```

Comment:
All messages in one? 

messages.foreach(message => send(properties, tableName, message) )